playbooks/roles/deploy-osh/tasks/main.yml

```diff
@@ -417,3 +417,19 @@
   tags:
     - heat
     - run
+
+- name: Deploy Barbican
+  import_tasks: component-install.yml
+  vars:
+    _component_runcommand: "./tools/deployment/multinode/160-barbican.sh"
+    _component_run_env:
+      OSH_EXTRA_HELM_ARGS_HEAT: >
```

Comment:
same as my tempest change, copy&paste gone wrong :P

Reply:
I actually fixed it locally, only forgot to push :-)